addons/web/static/src/js/views/list/list_editable_renderer.js

```diff
@@ -966,8 +1006,28 @@ ListRenderer.include({
         if (this.addTrashIcon) {
             $thead.find('tr').append($('<th>', {class: 'o_list_record_remove_header'}));
         }
+        this.hasBeenResized = false;
         return $thead;
     },
+    /**
+     * Overriden to add a resize handle in editable list column headers.
+     * Only applies to data column headers
+     * 
+     * @override
+     * @private
+     */
+    _renderHeaderCell: function () {
+        const $th = this._super.apply(this, arguments);
+        const name = $th.data('name');
+        if (this.editable && this._hasVisibleRecords(this.state) && name && name !== 'sequence') {
```

Comment:
I don't know if we want to handle those special cases here (there are probably other widgets for which resizing is pointless). We may probably let the user do it if he wants (or is there a real issue with it?).